### DISCUSSION TITLE
[Feature] Share individual pictures

### DISCUSSION THREAD

**Discussion by bartowski1182** [immich-app/immich#2764] [Feature Request]
### The feature

Would be nice if instead of only being able to share albums you could also make a public link to individual pictures

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
You can do this by selecting a picture in the main gallery view.

**Reply by bartowski1182** [reply]
Ah I see what you mean, but it shares it more like an album with an individual item.. Maybe I'll work on a way to make a single image album share page look nicer and more like how google photos did it. I think also the confusion is from only being able to share from web so i glanced over that
